presto-hive/src/main/java/io/prestosql/plugin/hive/RetryDriver.java

```diff
@@ -31,10 +31,10 @@
 public class RetryDriver
 {
     private static final Logger log = Logger.get(RetryDriver.class);
-    private static final int DEFAULT_RETRY_ATTEMPTS = 10;
-    private static final Duration DEFAULT_SLEEP_TIME = Duration.valueOf("1s");
-    private static final Duration DEFAULT_MAX_RETRY_TIME = Duration.valueOf("30s");
-    private static final double DEFAULT_SCALE_FACTOR = 2.0;
+    public static final int DEFAULT_RETRY_ATTEMPTS = 10;
```

Comment:
```suggestion
    public static final int DEFAULT_MAX_ATTEMPTS = 10;
```

(https://github.com/prestosql/presto/pull/240#discussion_r257225977)

Reply:
Agreed, the original name was incorrect.

Reply:
ok